src/api/search/server.js

```diff
@@ -0,0 +1,11 @@
+const { Satellite } = require('@senecacdot/satellite');
```

Comment:
Weird, 1.5.1 exists for sure https://www.npmjs.com/package/@senecacdot/satellite

Reply:
Just updated, need to commit.